### PR TITLE
PLU-97: Add observability

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem

User is unaware of new actions added.

## Solution

Add a new optional field `description` to `IApp` interface, if the app has a description, it would default to be a "new" app.
Sort the app options to show the "new" app above.
Renamed `toolbelt` to `toolbox`

Screenshots:

https://github.com/opengovsg/plumber/assets/65110268/9570fb5a-14ca-431d-ba4b-f4a1f619abe1

## Tests
Check that pipe can still be published and works with if-then action.

### CODE REVIEW

File: packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx [hunk 1]
```diff
@@ -79,6 +81,13 @@ function ChooseAppAndEventSubstep(
   const apps: IApp[] = data?.getApps?.filter((app: IApp) =>
     isTrigger ? !!app.triggers?.length : !!app.actions?.length,
   )
+
+  apps.sort((a, b) => {
+    if (a.description) {
+      return -1
```
Comment (ogp-weeloong) [inline packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx:87]: if there are multiple apps with descriptions, do we also want to sort alphabetically by name within this group of apps?

File: packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx [hunk 2]
```diff
@@ -200,6 +209,31 @@ function ChooseAppAndEventSubstep(
...
+                }}
+              >
+                <Flex flexDir="column">
+                  <Flex gap={2} alignItems="center">
+                    <Text>{option.label}</Text>
+                    {option.description && (
+                      <Badge
+                        bgColor="interaction.muted.main.active"
+                        color="primary.600"
+                        padding="0.25rem 0.5rem"
```
Comment (ogp-weeloong) [inline packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx]:
```suggestion
                        px={2}
                        py={1}
```

In general as a team coding style we prefer using chakra units as much as possible

**Review by ogp-weeloong — APPROVED** [review]
LGTM in general

File: packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx (hunk 1 above)
Comment (ogp-weeloong) [reply packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx:87]: nvm discussed; we will only have 1 app with description for now

File: packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx (hunk 2 above)
Comment (m0nggh) [reply packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx]: okayyyy thksss

**Pull request merged**